build.sbt

```diff
@@ -312,15 +335,36 @@ lazy val stagedRelease = (project in file("core/src/test"))
 
 import ReleaseTransformations._
 
-// Don't use sbt-release's cross facility	
+// Don't use sbt-release's cross facility
 releaseCrossBuild := false
 
-def crossReleaseStep(step: ReleaseStep): Seq[ReleaseStep] = {
+lazy val downVersionPySpark = taskKey[Unit]("Replace PySpark")
+downVersionPySpark := {
+  "./downversion-pyspark.sh" !
+}
+
+lazy val updateCondaEnv = taskKey[Unit]("Update Glow Env To Latest Version")
+updateCondaEnv := {
+  "conda env update -f python/environment.yml" !
+}
+
+def crossReleaseStep(step: ReleaseStep, test: Boolean): Seq[ReleaseStep] = {
```

Comment:
Might be more clear to call this parameter "requiresPyspark" or "updatePysparkVersion"

Reply:
Done